tests/test_templates_layers.py

```diff
@@ -462,3 +465,69 @@ def test_random_layer_weights(self, n_subsystems, tol):
         params = [q.parameters for q in rec.queue]
         params_flat = [item for p in params for item in p]
         assert np.allclose(weights.flatten(), params_flat, atol=tol)
+
+
+class TestSimplifiedTwoDesign:
+    """Tests for the SimplifiedTwoDesign method from the pennylane.templates.layers module."""
+
+    @pytest.mark.parametrize("n_wires, n_layers, shape_weights", [(1, 2, (0,)),
+                                                                  (2, 2, (2, 1, 2)),
+                                                                  (3, 2, (2, 2, 2)),
+                                                                  (4, 2, (2, 3, 2))])
+    def test_circuit_queue(self, n_wires, n_layers, shape_weights):
```

Comment:
Separated this into two tests: `test_circuit_queue_types` and `test_circuit_queue_parameter_values`, could make errors be more explicit.

Reply:
Good point, I also sneaked in the split for the BasicEntangler tests.